scalene/scalene_statistics.py

```diff
@@ -32,7 +32,7 @@ class ScaleneStatistics:
     #
     def __init__(self) -> None:
         # time the profiling started
-        self.start_time: float = None
+        self.start_time: float = 0
```

Comment:
Leaving it at `None` would help detect cases where `start_clock` wasn't called.

Reply:
This is true, but it currently fails with an exception, _and_ it would force us to use an `Optional` type. I'd rather handle it without breaking the typesystem.

Reply:
How about an assertion that `start_time != 0`?  Otherwise we'll silently use a large elapsed time should `start_time` not be invoked.

Reply:
It honestly should never happen, so I guess an assert is the right choice. (I don't know why or how it happened for the mentioned issue.)